shell/packages/sandstorm-accounts-ui/account-settings.js

```diff
@@ -139,7 +139,15 @@ Template.sandstormAccountSettings.events({
       instance._profileSaved.set(true);
     });
   },
-  "change": function () { Template.instance()._profileSaved.set(false); },
+  "change": function (event, instance) {
+    if (event.target == instance.find("input[name='picture']")) { return; }
+    // Pictures get saved right away.
```

Comment:
Usually we place comments describing code above the code (unlike comments describing declarations).